Automattic/vip-go-mu-plugins
# Offload bulk indexing to queue

## Description

Adds cron jobs to process the index queue and offloads bulk operations to the queue

## Checklist

Please make sure the items below have been covered before requesting a review:

- n/a This change has relevant documentation additions / updates (if applicable).

## Steps to Test

1. Enable async indexing with `define( 'VIP_SEARCH_ENABLE_ASYNC_INDEXING', true );`
1. Edit a category that is attached to multiple posts
1. Look in the `wp_vip_search_index_queue` table
1. Note 1 job per post from the category, marked as `scheduled`
1. Ensure WP cron is running
1. Once the scheduled cron event runs, the posts will be removed from the queue table (b/c they have been processed)
